src/main/java/com/openshift/restclient/ClientBuilder.java

```diff
@@ -33,6 +33,7 @@
 import com.openshift.restclient.http.IHttpConstants;
 import com.openshift.restclient.utils.SSLUtils;
 
+import okhttp3.Authenticator;
```

Comment:
Yeah I'm not found about having to expose the okhttp API. +1 on having an abstraction